models/load_dc_people.py

```diff
@@ -1,12 +1,34 @@
 """Load knowhys."""
 
+import re
+from typing import Any
+
 from bs4 import BeautifulSoup
 from langchain.schema.document import Document
 
 from models.load_utils import clean
 from models.load_utils import to_markdown
 
 
+def clean_text(markdown_content: str) -> Any:
+    """Remove unwanted texts from dc people content."""
+    # init return text
+    text_data = ""
+
+    # Search for the position of "abstract" (case insensitive)
+    text_match = re.search(r"[(\d+)](#t(\d+)).", markdown_content, re.IGNORECASE)
```

Comment:
Again, the special characters in this regular expression need to be escaped. Your regular expression will match `(#t1)` anywhere in the text. In order to match `[1](#t1).` at the start of a line, it should be
`text_match = re.search(r"^\[\d+\]\(#t\d+\)\.", markdown_content, re.IGNORECASE | re.MULTILINE)`